labs/README.md

```diff
@@ -0,0 +1,48 @@
+# Golden Path - Creating and deleting Flux config for an application
+
+[Golden Path tutorial - Java](https://backstage.platform.hmcts.net/catalog/default/component/golden-path-java) - VPN needed.
+
+## Creating the flux config for your lab application 
+
+Inside the labs directory run the [`create-lab-flux-config.sh`](./create-lab-flux-config.sh) script to create the flux config needed for your lab application. Use the component name you used when creating your app with backstage, which should be your GitHub username and specify the type of application you are deploying.
+
+<details>
+<summary>Command to create the flux configuration for a Java app</summary>
```

Comment:
just have them expanded, people often miss these and for something so short it's fine